databuilder/extractor/athena_metadata_extractor.py

```diff
@@ -45,14 +45,14 @@ class AthenaMetadataExtractor(Extractor):
 
     def init(self, conf: ConfigTree) -> None:
         conf = conf.with_fallback(AthenaMetadataExtractor.DEFAULT_CONFIG)
-        self._cluster = '{}'.format(conf.get_string(AthenaMetadataExtractor.CATALOG_KEY))
+        self._cluster = conf.get_string(AthenaMetadataExtractor.CATALOG_KEY)
 
         self.sql_stmt = AthenaMetadataExtractor.SQL_STATEMENT.format(
             where_clause_suffix=conf.get_string(AthenaMetadataExtractor.WHERE_CLAUSE_SUFFIX_KEY),
             catalog_source=self._cluster
         )
 
-        LOGGER.info('SQL for Athena metadata: {}'.format(self.sql_stmt))
+        LOGGER.info('SQL for Athena metadata: %s', self.sql_stmt)
```

Comment:
I believe we should be using f here as well for consistency

Reply:
@allisonsuarez string.format or f-string is eager evaluation, which mean that the logging message still evaluate event it's not logged to output. So in somehow, it's better.
What do you think?